doc/user/content/self-managed/installation/install-on-aws.md

```diff
@@ -14,151 +23,309 @@ For testing purposes only. For testing purposes only. For testing purposes only.
 
-See the
-[README](https://github.com/MaterializeInc/terraform-aws-materialize/blob/main/README.md)
-for information on how to deploy the infrastructure.
+{{< warning >}}
 
-### `kubectl`
+The sample Terraform module is for **evaluation purposes only** and not intended
+for production use. It is provided to help you get started with Materialize for
+evaluation purposes only. Materialize does not support nor recommends this
+module for production use. Materialize does not guarantee tests for changes to
+the module.
 
-Install `kubectl` and configure cluster access. For details, see the [Amazon EKS
-documentation](https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html).
+For simplicity, this tutorial stores your RDS Postgres secret in a file. In
+practice, refer to your organization's official security and
+Terraform/infrastructure practices.
 
-Configure `kubectl` to connect to your EKS cluster, replacing
-`<your-region>` with the region of your EKS cluster:
+{{< /important >}}
 
-```bash
-aws eks update-kubeconfig --name materialize-cluster --region <your-region>
-```
+1. If you do not have Terraform installed, [install
+   Terraform](https://developer.hashicorp.com/terraform/install?product_intent=terraform).
 
-{{< note >}}
+1. Clone or download the [Materialize's sample Terraform
+   repo](https://github.com/MaterializeInc/terraform-aws-materialize).
 
-The exact authentication method may vary depending on your EKS configuration.
+1. Go to the Materialize Terraform repo directory.
 
-{{< /note >}}
+   ```bash
+   cd terraform-aws-materialize
+   ```
 
-To verify, run the following command:
+1. Copy the `terraform.tfvars.example` file to `terraform.tfvars`.
 
-```bash
-kubectl get nodes
-```
+   ```bash
+   cp terraform.tfvars.example terraform.tfvars
+   ```
 
-For help with `kubectl` commands, see [kubectl Quick
-reference](https://kubernetes.io/docs/reference/kubectl/quick-reference/).
+1. Edit the `terraform.tfvars` file to set the values for your AWS environment.
+   In particular,
 
-### Helm 3.2.0+
+   - set your `database_password` to a secure password.
+   - set your `node_group_ami_type` to a `"AL2023_ARM_64_STANDARD"`.
+   - set your `node_group_instance_types` to a supported instance type for
+     ARM64.
 
-If you don't have Helm version 3.2.0+ installed, refer to the [Helm
-documentation](https://helm.sh/docs/intro/install/).
 
 
-## A. Install the Materialize Operator
+   ```bash
+   cluster_name      = "my-test-eks"
+   environment       = "my-test"
+   vpc_name          = "my-test-vpc"
+
+   bucket_name       = "my-test-bucket"
+   db_identifier     = "my-test-db"
+   database_password = "enter-your-secure-password"
+
+   tags = {
+     Environment = "my-test"
+     Team        = "my-test-team"
+     Project     = "my-test-project"
+   }
+
+   node_group_ami_type       = "AL2023_ARM_64_STANDARD"
+   node_group_instance_types = ["r6g.medium"]
+   node_group_desired_size   = 3
```

Comment:
can make these more inline with the values in examples/simple.

Reply:
Actually, I do need to make these like those.  Otherwise, `Insufficient cpu, 3 Insufficient memory. `